lib/tasks/fake_data.rake

```diff
@@ -12,7 +12,7 @@ namespace :fake_data do
       person = Person.create!(
         first_names: Faker::Name.first_name,
         last_name: Faker::Name.last_name,
-        date_of_birth: Faker::Date.between(from: 80.years.ago, to: 20.years.ago),
+        date_of_birth: Faker::Date.between(from: 80.years.ago.to_date, to: 20.years.ago.to_date),
```

Comment:
are the extra `.to_date`s necessary ? Faker seems to run fine without them:
```
pry(main)> Faker::Date.between(from: 80.years.ago, to: 20.years.ago)
=> Fri, 27 Oct 1950
```

Reply:
I just added it because I got a warning `Incompatible types Required:Date Passed:Time` on my pre-commit checks
But the project's rubocop config seems to ignore it. 

So I guess we can remove it.

Reply:
lets go with the project's rubocop rules